space-time.Rmd

```diff
@@ -1,13 +1,15 @@
-# Space and time {#space-time}
+# Maps and surfaces {#space-time}
 
 ```{r, include = FALSE}
 source("common.R")
 columns(1, 2 / 3)
 ```
 
-## Surface plots {#surface}
+Up to this point the visualisation problems discussed have focused on cases where data can be suitably represented by geoms such as points and lines which tend to represent data in a straightforward fashion. We have also considered cases where more complex geoms such as densities, violins and boxplots can be used to introduce some degree of abstraction. What has not yet been considered are cases where the a three-dimensional (3d) visualisation is required (as in a surface plot) or where data need to be overlaid on a two dimensional (2d) geospatial plot (a map). These visualisation situations require somewhat different tools, and are the focus of this chapter.
 
-ggplot2 does not support true 3d surfaces. However, it does support many common tools for representing 3d surfaces in 2d: contours, coloured tiles and bubble plots. These all work similarly, differing only in the aesthetic used for the third dimension. Here is an example of a contour plot: \index{Surface plots} \index{Contour plot} \indexf{geom\_contour} \index{3d}
+## Surfaces {#surface}
+
+The ggplot2 package does not support true 3d surfaces. However, it does support many common tools for representing 3d surfaces in 2d: contours, coloured tiles and bubble plots. These all work similarly, differing only in the aesthetic used for the third dimension. Here is an example of a contour plot: \index{Surface plots} \index{Contour plot} \indexf{geom\_contour} \index{3d}
```

Comment:
Footnote pointing to rayshader ggplot2 support?

Also, if you're moving indexes around, my current workflow is to always put them under the heading (this is technically mildly incorrect, but it makes it easier to ensure that every subsection has index entries, when we do a pass through just prior to publication)